pandas/tests/series/test_sorting.py

```diff
@@ -64,6 +64,24 @@ def test_sort_values(self):
         ordered = ts.sort_values(ascending=False, na_position='first')
         assert_almost_equal(expected, ordered.valid().values)
 
+        # ascending=[False] should behave the same as ascending=False
+        ordered = ts.sort_values(ascending=[False])
+        expected = np.sort(ts.valid().values)[::-1]
```

Comment:
sorry didn't notice this. Can you construct the result series and use ``tm.assert_series_equal`` for both of these. (literally construct it, don't sort things)